src/collections/_documentation/platforms/python/index.md

```diff
@@ -2,55 +2,398 @@
+
+If you wish to append information, thus making the grouping slightly less aggressive, you can do that as well by adding the special string `{{default}}` as one of the items.
+
+For more information, see [Aggregate Errors with Custom Fingerprints](https://blog.sentry.io/2018/01/18/setting-up-custom-fingerprints).
+
+## Advanced Usage
+
+### Advanced Configuration
+
+The Sentry SDK sets the options when you first initialize the SDK.
+
+```python
+import sentry_sdk
+
+sentry_sdk.init(
+    'https://<key>@sentry.io/<project>',
+    max_breadcrumbs=50,
+    debug=True,
+)
+```
+
+For more information, see:
+
+- [Sentry’s complete list of Common Options across SDKs]({%- link _documentation/error-reporting/configuration/index.md -%})
+- [Full documentation on Environments]({%- link _documentation/enriching-error-data/environments.md -%})
+
+### Breadcrumbs
+
+Sentry will automatically record certain events, such as changes to the URL and XHR requests to provide context to an error.
+
+You can manually add breadcrumbs on other events or disable breadcrumbs.
+
+```python
+from sentry_sdk import add_breadcrumb
+
+add_breadcrumb(
+    category='auth',
+    message='Authenticated user %s' % user.email,
+    level='info',
+)
+```
+
+For more information, see:
+
+- [Full documentation on Breadcrumbs]({%- link _documentation/enriching-error-data/breadcrumbs.md -%})
+- [Debug Issues Faster with Breadcrumbs](https://blog.sentry.io/2016/05/04/breadcrumbs).
+
+### Filter Events & Custom Logic
+
+Sentry exposes a beforeSend callback which can be used to filter out information or add additional context to the event object.
+
+```python
+import sentry_sdk
+
+def strip_sensitive_data(event, hint):
+    # modify event here
+    return event
+
+sentry_sdk.init(
+    before_send=strip_sensitive_data
+)
+```
+
+For more information, see:
+
+- [Full documentation on Filtering Events]({%- link _documentation/error-reporting/configuration/filtering.md -%})
+- [Manage Your Flow of Errors Using Inbound Filters](https://blog.sentry.io/2017/11/27/setting-up-inbound-filters).
+
+### Capturing Messages
+
+Typically, the Sentry SDK does not emit messages. A message is just some textual information that should be sent to Sentry. This is most useful when you’ve overridden fingerprinting but need to give a useful message.
+
+```python
+from sentry_sdk import capture_message
+
+capture_message('Something went wrong')
+```
+
+### Integrations
+
+*Integrations* extend the functionality of the SDK for some common frameworks and libraries. Similar to plugins, they extend the functionality of the Sentry SDK. A call to `sentry_sdk.init` configures integrations. Unless you set `default_integrations` to `False`, Sentry automatically adds any default integration not in the list below.
+
+#### **Web Frameworks**
```

Comment:
This seems like a formatting bug (declaring a h4 and then also making it bold)

If we want to make `h4` look different we should change the CSS IMO